test/suites/test_dbapi.py

```diff
@@ -50,6 +51,47 @@ def tearDownClass(self):
         self.srv.stop()
         self.srv.clean()
 
+    @skip_or_run_sql_test
+    def test_None(self):
+        super(TestSuite_DBAPI, self).test_None()
```

Comment:
It is hard to do on per suite class, correct? If so, I don't mind: just re-verified it.

Reply:
As far I understand, not all tests from DatabaseAPI20Test verify SQL-related stuff. It looks like some tests verify other things because they work against tarantool 1.10. So we cannot wrap the whole class by this decorator to skip all tests.

Reply:
That's looks unexpected. Or, okay, something may work if it does not use SQL requests in the binary protocol. However such functionality without ability to execute an SQL request looks useless. Can you show an example?

Reply:
For example, this [test](https://github.com/baztian/dbapi-compliance/blob/v1.15.0/dbapi20.py#L116).

Reply:
Okay, got it. As for me, it is quite implementation specific. If it is technically possible, I would skip all the database API test suite on 1.10. If it is hard to implement, let it be, but leave a comment: why it is skipped on per test basis and why some tests are not skipped (or skip them too).

Reply:
It's not hard to implement. Ok, skipping the whole test suite.